### ISSUE TITLE
[HOLD for payment 2025-01-13] [HOLD for payment 2025-01-07] Chat - App crashes when creating group chat offline and returning online on report details RHP

### ISSUE THREAD

**Issue by mitarachim** [Expensify/App#54595]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.79-0
**Reproducible in staging?:** Yes
**Reproducible in production?:** No
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:** N/A
**If this was caught during regression testing, add the test name, ID and link from TestRail:** Exp
**Email or phone of affected tester (no customers):** applausetester+kh1012032@applause.expensifail.com
**Issue reported by:** Applause Internal Team
**Device used:** Mac 15.0 / Chrome
**App Component:** Chat Report View

## Action Performed:
1. Go to staging.new.expensify.com
2. Go offline.
3. Open FAB > Start chat.
4. Create a group chat with any user.
5. Click on the group chat header.
6. Go online.

## Expected Result:
App will not crash.

## Actual Result:
App crashes when creating group chat offline and returning online on report details RHP.

## Workaround:
Unknown

## Platforms:
- [ ] Android: Standalone
- [ ] Android: HybridApp
- [ ] Android: mWeb Chrome
- [x] iOS: Standalone
- [x] iOS: HybridApp
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos

https://github.com/user-attachments/assets/ef70a051-0aaf-4298-84a1-a697bb0d2720

[Bug6702687_1735252105899!crash.txt](https://github.com/user-attachments/files/18258218/Bug6702687_1735252105899.crash.txt)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Issue Owner</summary>Current Issue Owner: @sonialiap</details>

**Comment by thelullabyy** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
App crashes when creating group chat offline and returning online on report details RHP.

### What is the root cause of that problem?
It caused by [this PR](https://github.com/Expensify/App/pull/54247), we don't add 
### What changes do you think we should make in order to solve the problem?


### What specific scenarios should we cover in automated tests to prevent reintroducing this issue in the future?



https://github.com/Expensify/App/blob/e94df0f79443f99b33be867fec8637c6a9aa5821/src/pages/ReportDetailsPage.tsx#L90

We need to add default value and use ?. access to this line
### What alternative solutions did you explore? (Optional)



**Reminder:** Please use plain English, be brief and avoid jargon. Feel free to use images, charts or pseudo-code if necessary. Do not post large multi-line diffs or write walls of text. Do not create PRs unless you have been hired for this job.

**Comment by jasperhuangg** [comment]
Hey @TMisiukiewicz can you please fix this regression that was caused by your PR?

**Comment by TMisiukiewicz** [comment]
@jasperhuangg sure, I'm on it

**Comment by TMisiukiewicz** [comment]
Opened a PR

**Comment by Beamanator** [comment]
Helping review this one!

**Comment by isagoico** [comment]
issue not reproducible in the latest stg build 🎉 

https://github.com/user-attachments/assets/bb86b547-594d-4b2b-917f-157d456263ca

**Comment by Beamanator** [comment]
Mainly correct, except there was a proposal here: https://github.com/Expensify/App/issues/54595#issuecomment-2563402103

So @TMisiukiewicz would you please let us know if you used that proposal in your solution? It looks like a pretty straightforward fix so just checking if any payout is needed for that contrib.

**Comment by TMisiukiewicz** [comment]
@Beamanator yeah it was pointing the right place where changes were needed so this proposal was helpful

**Comment by Beamanator** [comment]
Cool thanks!!

Ok so @sonialiap I believe @thelullabyy is due something for helping narrow down where the bug was coming from - I think we normally go with a quarter of the standard fix price which would be $62.5, but you can confirm 🙏

**Comment by sonialiap** [comment]
Sounds good, thanks!

**Comment by sonialiap** [comment]
Payment summary:
- @thelullabyy $62.5 for proposal - offer sent in [upwork](https://www.upwork.com/nx/wm/offer/105610809)

**Comment by sonialiap** [comment]
@thelullabyy please accept the proposal in Upwork. Let me know if you have any questions!

**Comment by sonialiap** [comment]
Thanks, paid!
